<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="blackRadioBtn">
       <property name="text">
        <string>Black</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="blueRadioBtn">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="redRadioBtn">
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>Blod</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Underline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
